modules/security/pages/login-password.adoc

```diff
@@ -0,0 +1,22 @@
+= Complexity password policy and Login protection
+:description: Login, Password
+:pp: {plus}{plus}
+:page-aliases: README.adoc, readme.adoc
+
+Tigergraph provides complexity password policy and login protection to improve the security.
+
+== Password Policy
+
+The password policy is a set of rules and requirements that dictate how passwords should be created, password expiration times, and password rotation rules to ensure security.
+See xref:tigergraph-server:security:password-policy[] for a detailed description
+
+== Login Protection
+
+Login protection is a security feature that helps safeguard user accounts from unauthorized access, enhancing overall account security.
+See xref:tigergraph-server:security:login-protection[] for a detailed description
```

Comment:
```suggestion
See xref:login-protection[] for a detailed description
```

Reply:
If referencing a page within the same component/repo, do not give the name of the component.
The reason why is: referencing the component with jump to the newest version of that component (if you didn't specify a version).